Sources/Workflow/Protocols/OrchestrationResponder.swift

```diff
@@ -32,6 +32,13 @@ public protocol OrchestrationResponder {
      - Parameter onFinish: A closure that is executed when the responder is finished abandoning.
      */
     func abandon(_ workflow: AnyWorkflow, onFinish: (() -> Void)?)
+    /**
+     Respond to the `Workflow` completing.
+     - Parameter workflow: The `AnyWorkflow` that is being completed.
+     - Parameter passedArgs: The `AnyWorkflow.PassedArgs` to be passed to `onFinish`.
+     - Parameter onFinish: A closure that is executed when the responder is finished completing.
+     */
+    func complete(_ workflow: AnyWorkflow, passedArgs: AnyWorkflow.PassedArgs, onFinish: ((AnyWorkflow.PassedArgs) -> Void)?)
```

Comment:
I don't see it as a conversation to resolve so I'm adding here.
1. We need to generate the jazzy docs for this change
2. We need to version ramp the minor version of the app as we are adding Complete to the API.

Reply:
I see Release CI has been bumping the patch version on merges to main, how do we increment the minor spot?

Reply:
I have version bumped the minor with `bundle exec fastlane minor`

Reply:
Last commit generated the docs and added some more.